pom.xml

```diff
@@ -123,6 +123,7 @@
     <asm.bundle.version>8.0.1</asm.bundle.version>
     <jackson.bundle.version>${jackson.version}</jackson.bundle.version>
     <slf4j.version>2.0.11</slf4j.version>
+    <log4j.version>2.22.0</log4j.version>
```

Comment:
We'll eventually want to unify the logging system

Reply:
Note that this is just for tests

Reply:
Noted, but the idea is to keep dependencies to the minimum. Anyway, this is not urgent or critical.